src/ansys/aedt/core/hfss.py

```diff
@@ -5919,6 +5919,104 @@ def get_antenna_data(
             self.logger.error("Farfield solution data could not be exported.")
             return False
 
+    @pyaedt_function_handler()
+    def get_rcs_data(
+        self,
+        frequencies=None,
+        setup=None,
+        expression="ComplexMonostaticRCSTheta",
+        variations=None,
+        overwrite=True,
+        link_to_hfss=True,
+        variation_name=None,
+    ):
+        """Export the radar cross-section data and return an
+        instance of the ``RcsSolutionDataExporter`` object.
```

Comment:
```suggestion
        """Export the radar cross-section data.
        This method returns an instance of the ``RcsSolutionDataExporter`` object.
```
Description titles should be short.